main.js

```diff
@@ -85,12 +68,69 @@ function makeNewActivity(event) {
   )
 }
 
+function validateInputs() {
+  validateDescription(); // had to do function for each otherwise it wouldn't display all of them
+  validateMinutes();
+  validateSeconds();
+  validateCategory();
+}
+
+function checkInputs() {
+  if (validateCategory() || validateDescription() || validateMinutes() || validateSeconds()) {
+    console.log('falseTest');
+    return false;
+  } console.log('trueTest');
+  return true;
+}
+function validateCategory() {
+ if (currentActivity.category === "") {
+   displayErrorMessage('category');
+   return true;
+ }
+}
+
+function validateDescription() {
+ if (currentActivity.description === "") {
+   displayErrorMessage('description');
+   changeInputColor('description');
+   return true;
+ }
+}
+
+function validateMinutes() {
+ if (currentActivity.minutes === "") {
+   displayErrorMessage('minutes');
+   changeInputColor('minutes');
+   return true;
+ }
+}
+
+function validateSeconds() {
+ if (currentActivity.seconds === "") {
+   displayErrorMessage('seconds');
+   changeInputColor('seconds');
+   return true;
+ }
+}
+
+function displayErrorMessage(section) {
+  document.querySelector(`.${section}-error`).classList.remove("hidden");
+} //Do for each!
+
+function changeInputColor(section) {
+  document.querySelector(`.${section}-input`).classList.add("input-error-color")
+}
+
+function toggleElement(className1) {
+  document.querySelector(`.${className1}`).classList.toggle("hidden");
+}
+
 function getCategory(parent) {
   for (var i = 0; i < parent.children.length; i++) {
     if (!parent.children[i].classList.contains("btn-default")) {
       return parent.children[i].innerText;
     }
-  }
+  } return "";
 }
 
```

Comment:
Let's break that return on to the next line (134) instead of next to a curly 𝕭racket.